sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -425,18 +428,18 @@ def extract_output(self, accumulator):
 
```

Comment:
Probably we add some documentation on TupleCombineFn and SingleInputTupleCombineFn classes.